pytest.ini

```diff
@@ -24,5 +24,6 @@
 
 [pytest]
 pep8ignore = docs/conf.py ALL
-addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_search --cov-report=term-missing docs tests invenio_search
+addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_search --cov-report=term-missing
+testpaths = docs tests invenio_search
```

Comment:
This solution seems even better than inveniosoftware/cookiecutter-invenio-module#110 👍 
We can update the troubleshooting issue inveniosoftware/troubleshooting#17

Reply:
I got it from cookiecutter actually: https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/pytest.ini#L5

We should probably propagate these to other Invenio modules as well, but I believe this is a discussion/action that is ongoing in the current Invenio sprint.